src/main/java/org/broadinstitute/hellbender/utils/config/GATKConfig.java

```diff
@@ -0,0 +1,135 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import org.aeonbits.owner.Accessible;
+import org.aeonbits.owner.Mutable;
+import org.aeonbits.owner.Config.LoadPolicy;
+import org.aeonbits.owner.Config.LoadType;
+import org.aeonbits.owner.Config.Sources;
+
+import java.util.List;
+
+/**
+ * Configuration file for GATK options.
+ */
+@LoadPolicy(LoadType.MERGE)
+@Sources({
+        "file:${" + GATKConfig.CONFIG_FILE_VARIABLE_NAME + "}",                         // Variable for file loading
+        "file:GATKConfig.properties",                                                   // Default path
+        "classpath:org/broadinstitute/hellbender/utils/config/GATKConfig.properties"    // Class path
+})
+public interface GATKConfig extends Mutable, Accessible {
+
+    // =================================================================================================================
+    // =================================================================================================================
+    // Meta Options:
+    // =================================================================================================================
+    // =================================================================================================================
+
+    String CONFIG_FILE_VARIABLE_NAME = "pathToGatkConfig";
+
+    // =================================================================================================================
+    // =================================================================================================================
+    // System Options:
+    // =================================================================================================================
+    // =================================================================================================================
+
+    String SYSTEM_PROPERTY_PREFIX = "system.";
+
+    // ----------------------------------------------------------
+    // General Options:
+    // ----------------------------------------------------------
+
+    @Key("system.gatk_stacktrace_on_user_exception")
+    @DefaultValue("true")
+    boolean gatk_stacktrace_on_user_exception();
```

Comment:
these are named weirdly for java methods.  Typically we'd do something like getGatkStacktraceOnUserException